biothings_explorer/call_apis_v2/helpers.py

```diff
@@ -0,0 +1,6 @@
+import yaml
+
+
+def yaml_2_json(_input):
+    doc = yaml.load(_input)
```

Comment:
@sengineer0 don't know which version you use, my `yaml.load` complains here "missing Loader".

Reply:
I used `PyYAML==5.4.1` as pined in the requirements.txt
@newgene

Reply:
I see, let's use the latest.